### PR TITLE
EVG-1496 Surface host count on TaskQueue page

### PR DESCRIPTION
**Pull request by minnakt**
[EVG-1496](https://jira.mongodb.org/browse/EVG-1496)

### Description 
This PR surfaces the host count on the `/task-queue` page for each distro. Previously, this would only show the number of tasks on the queue for each distro.

Additionally, this PR adds a `SelectSearch` component which allows users to search for specific distros and select them from a dropdown (see mockups attached to JIRA ticket). Leafygreen is currently working on adding this functionality to the `Select` component so eventually this component should be able to be entirely deleted.

### Screenshots
(Thank you Sophie for showing me how to do this 😌)

Screen Shot 2021-11-22 at 9 13 54 PM
Screen Shot 2021-11-22 at 9 13 50 PM

### Testing 
- Add to storybook `SelectSearch.stories.tsx`
- Unit tests in `SelectSearch.test.tsx`
- Edited e2e test for task queue page

### Evergreen PR 
- evergreen-ci/evergreen/pull/5229

### CODE REVIEW

**Comment by minnakt** [comment]
The e2e tests will fail until the backend PR is merged

File: src/components/SelectSearch/SelectSearch.test.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+const RenderSelectSearch = (props: any) => (
+  <SelectSearch
+    label="Just a test"
+    searchPlaceholder="Test Placeholder"
+    {...props}
+  />
+);
+
+describe("SelectSearch", () => {
+  test("Should toggle dropdown when clicking on it ", () => {
```
Comment (SupaJoon) [inline src/components/SelectSearch/SelectSearch.test.tsx:15]:
```suggestion
  test("Should toggle dropdown when clicking on it", () => {
```

File: src/components/SelectSearch/index.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
+import { useState, PropsWithChildren, useRef, useEffect, useMemo } from "react";
```
Comment (SupaJoon) [inline src/components/SelectSearch/index.tsx:1]: We may be able to modify SearchableDropdown and to match the design instead of introducing this component

**Comment by khelif96** [comment]
I replaced the SelectSearch component with the SearchableDropdown component 
image

**Review by khelif96 — APPROVED** [review]

**Review by malikchaya2 — APPROVED** [review]

**Pull request merged**